src/port-to-eval-frame.js

```diff
@@ -1,11 +1,11 @@
 /* global IODIDE_EVAL_FRAME_ORIGIN  */
 
 import Mousetrap from "mousetrap";
-import { store } from "./store";
 import { addLanguage, setKernelState } from "./actions/actions";
 import { genericFetch as fetchFileFromServer } from "./tools/fetch-tools";
 import evalQueue from "./actions/evaluation-queue";
 import validateActionFromEvalFrame from "./actions/eval-frame-action-validator";
+import messagePasserEd from "./redux-to-port-message-passer";
```

Comment:
can this be `messagePasserEditor` or something? the `Ed` took me a few seconds, and would probably be confusing to a new contributor

Reply:
yep that seems sensible -- i'll make the string change and merge